hash/zx81_cass.xml

```diff
@@ -2,21 +2,32 @@
 <!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
 <!--
 license:CC0-1.0
-
-Sinclair ZX81 Cassettes
-
-This list was compiled from the archive at http://www.zx81stuff.org.uk/. Updated to 22/08/2016
```

Comment:
This comment served a purpose, the site is the main source for this softlist and the date tells us which site update the softlist was last synced with.

Reply:
Ok, so I've just created a list of missing dumps which includes all entries published since the latest softlist update, up to the last site update three years ago. I think this is more in line with what's currently being done in other softlists?